pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-circulation-storage</artifactId>
   <groupId>org.folio</groupId>
-  <version>12.2.0-SNAPSHOT</version>
+  <version>12.2.1-SNAPSHOT</version>
```

Comment:
Why has the module version been changed?

Reply:
As in case with mod-patron, it's wrong updating of snapshot number, because doesn't make sense. Will fix it, thanks.